### PR TITLE
made the code simpler

### PR DESCRIPTION
**Pull request by mjrish96**

### CODE REVIEW

**Review by normalhuman — CHANGES_REQUESTED** [review]
Please follow the [development workflow](https://github.com/sympy/sympy/wiki/Development-workflow), in particular regarding pull request description, title, and creating a branch for each pull request. 

And testing. This line throws the error `TypeError: 'Symbol' object is not iterable`.

File: examples/beginner/basic.py [hunk 1]
```diff
@@ -9,9 +9,7 @@
 
 
 def main():
-    a = Symbol('a')
-    b = Symbol('b')
-    c = Symbol('c')
+    a,b,c = Symbol('a b c')
```
Comment (ylemkimon) [inline examples/beginner/basic.py]: `Symbol` returns single symbol, whereas `symbols` return multiple symbols if given.

**Comment by RISHAV98** [comment]
symbols module  cannot be import from sympy.

**Comment by smichr** [comment]
> symbols module cannot be import from sympy.

If you do a search for `import symbols` you will see how this is imported successfully.

**Comment by RISHAV98** [comment]
a,b,c = symbols('a b c')

**Comment by RISHAV98** [comment]
have a look on this

**Pull request closed without merging**